src/collectd.conf.pod

```diff
@@ -6849,6 +6849,22 @@ are available on the server side. I<Bytes> must be at least 1024 and cannot
 exceed the size of an C<int>, i.e. 2E<nbsp>GByte.
 Defaults to C<4096>.
 
+=item B<LowSpeedLimit> B<true|false>
+
+If set to B<true>, average transfer speed in bytes per second will be checked. 
+In case it is below B<LowLimitBytesPerSec> connection will be considered slow 
+and aborted.
+
+=item B<LowLimitBytesPerSec> I<Bytes>
+
+Sets bytes per second value for B<LowSpeedLimit> to make a decission if 
+connection is too slow. Default value is C<100>.
+
+=item B<PostTimeoutSec> I<Seconds>
+
+If defined, provided positive integer value will be used to set maximum time
```

Comment:
If defined, the positive integer value will set the maximum timeout in seconds of the HTTP POST operation.